### PR TITLE
Proxmox VE integration docs

### PR DESCRIPTION
**Pull request by erangaj**
This is the documentation for the Proxmox VE component.

### CODE REVIEW

File: source/_components/proxmox.markdown [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+In addition, the `proxmox` component adds switches to on/off virtual machines/containers.
+
+## {% linkable_title Configuration %}
+
+To use the `proxmox` component in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+proxmox:
+   host: 192.168.1.100
```
Comment (klaasnicolaas) [inline source/_components/proxmox.markdown]:
```suggestion
   host: IP_ADDRESS
```

File: source/_components/proxmox.markdown [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+   port: 8006
```
Comment (klaasnicolaas) [inline source/_components/proxmox.markdown]: Please remove `line 38`

File: source/_components/proxmox.markdown [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+   verify_ssl: False
```
Comment (klaasnicolaas) [inline source/_components/proxmox.markdown]: Please remove `line 39`

File: source/_components/proxmox.markdown [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+   username: user
```
Comment (klaasnicolaas) [inline source/_components/proxmox.markdown]:
```suggestion
   username: USERNAME
```

File: source/_components/proxmox.markdown [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+   password: mysupersecretpassword
```
Comment (klaasnicolaas) [inline source/_components/proxmox.markdown]:
```suggestion
   password: PASSWORD
```

File: source/_components/proxmox.markdown [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+   realm: pam
```
Comment (klaasnicolaas) [inline source/_components/proxmox.markdown]: Please remove `line 42`

File: source/_components/proxmox.markdown [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+   nodes:
```
Comment (klaasnicolaas) [inline source/_components/proxmox.markdown]: Please remove `line 43 - 45`

File: source/_components/proxmox.markdown [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+     - node1
+     - node2
+   vms:
```
Comment (klaasnicolaas) [inline source/_components/proxmox.markdown]: Please remove `line 46 - 49`

File: source/_components/proxmox.markdown [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+     - 101
+     - 102
+     - 205
+   start_stop_all_vms: False
```
Comment (klaasnicolaas) [inline source/_components/proxmox.markdown]: Please remove `line 50`

File: source/_components/proxmox.markdown [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+   start_stop_vms:
```
Comment (klaasnicolaas) [inline source/_components/proxmox.markdown]: Please remove `line 51 - 53`

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
There are a lot of changes for you in the `example config` section. Only required variables in the first example, you can add a more advanced example with optional variables.

File: source/_components/proxmox.markdown (hunk 2 above)
Comment (erangaj) [reply source/_components/proxmox.markdown]: Done. All the optional configs removed.

File: source/_components/proxmox.markdown (hunk 3 above)
Comment (erangaj) [reply source/_components/proxmox.markdown]: Done. All the optional configs removed.

File: source/_components/proxmox.markdown (hunk 6 above)
Comment (erangaj) [reply source/_components/proxmox.markdown]: Done. All the optional configs removed.

File: source/_components/proxmox.markdown (hunk 7 above)
Comment (erangaj) [reply source/_components/proxmox.markdown]: Done. All the optional configs removed.

File: source/_components/proxmox.markdown (hunk 8 above)
Comment (erangaj) [reply source/_components/proxmox.markdown]: Done. All the optional configs removed.

File: source/_components/proxmox.markdown (hunk 9 above)
Comment (erangaj) [reply source/_components/proxmox.markdown]: Done. All the optional configs removed.

File: source/_components/proxmox.markdown (hunk 10 above)
Comment (erangaj) [reply source/_components/proxmox.markdown]: Done. All the optional configs removed.

**Comment by erangaj** [comment]
Thank you. I have removed all the optional elements from the configuration.

**Comment by klaasnicolaas** [comment]
Parent PR still open?

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**